package.json

```diff
@@ -17,6 +17,6 @@
   },
   "homepage": "https://github.com/Mor-s-organization/misconfiguration-check#readme",
   "dependencies": {
-    "lodash": "^4.17.20"
+    "axios": "^0.19.2"
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2021-3749   
**Check Name:** nodejs-axios: Regular expression denial of service in trim function   
**Severity:** HIGH   
**Fixed Version:** 0.21.2   
**Description:** axios is vulnerable to Inefficient Regular Expression Complexity  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2021-3749